.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: java
+jdk: openjdk8
```

Comment:
Actually, you should use OpenJDK 11 with `--release 6`, imho. Running a genuine JDK 6 is not longer recommended (or supported on Travis CI, afaik).

Reply:
Ooh, thanks for bringing the relatively new `--release` option to my attention! That's cool. It takes care of setting the bootclasspath during compilation to ensure that the code being compiled uses only APIs that were available in the old release.

https://stackoverflow.com/questions/43102787/what-is-the-release-flag-in-the-java-9-compiler

If we're going to say that Prevayler 2.7 is still compatible with Java 6, it might still be nice to test it against an actual JDK 6 build, but `--release` might be sufficient given the low likelihood of functional regressions.

One catch is that `--release` was introduced in JDK 9, and I still would like to be able to compile and test on JDK 8 for my own convenience (because that happens to still be the JDK available on the Debian distribution supported on ChromeOS), so hopefully it can be done without changing our `pom.xml` in a way that breaks JDK 8 compatibility...

Reply:
There's no JDK 11 on ChromeOS, yet? What about using https://adoptopenjdk.net/releases.html?variant=openjdk11&jvmVariant=hotspot as an alternative?

Reply:
Yeah, I'm pretty sure I can download and install it, it's just not available in the Debian or Google apt-get repositories until the next Debian LTS release later this year. I'm just trying to minimize how much custom installation I do on the machine. But besides that, it serves as a reminder that Java 8 is still widely deployed, so we don't necessarily want to rush into requiring later versions. I'll update our project board for Prevayler 3.0 to target Java 11. :)

Reply:
Update: After some further experimentation with trying to get OpenJDK 6 to work on Travis CI, I came back around to your suggestion about using OpenJDK 11 with the `--release` option. To address my own use case, I was able to configure Maven "profiles" to use different compiler options for pre-1.9 and post-1.9 compilers.

See commit d9ce923302fe0f448fabb352f6dd1a89e608a594 and the build running here: https://travis-ci.org/jsampson0/prevayler

(I'm not entirely comfortable with the permissions that Travis CI asks for, so at the moment this is a distinct user account with its own fork.)